tests/test_taskipy.py

```diff
@@ -216,13 +216,13 @@ def setUp(self):
         # suppress resource warnings, as they are false positives caused by psutil
         warnings.simplefilter('ignore', category=ResourceWarning)
 
-    def interrupt_task(self, process: subprocess.Popen):
+    def interrupt_task(self, process: subprocess.Popen, sig: int = signal.SIGINT):
```

Comment:
We want to test that signals are passed from taskipy to the inner process, correct?

But `interrupt_task` actually looks for the pid of the **inner** process and sends the signal directly to it. This is because keyboard interrupts are actually sent into the inner process and not the outer one!

What we want is another method (we can call it `send_signal_to_task`) which sends the signal to taskipy's process. That way we can make sure that we actually test our signal handlers.

Hope that this was clear ;)

Reply:
I've tried this one: https://github.com/illBeRoy/taskipy/pull/19/files#diff-6eb2eb5bdad69abb5ba39aea93372bf6R259 but it doesn't work as expected. Only `self.interrupt_task(process, signal.SIGTERM)` works, dunno why :/

Reply:
glad that we got that figured out :)
In that case (in order to prevent mixups) - can you please revert the changes to this method?